installscripts/jazz-terraform-unix-noinstances/scripts/destroy.sh

```diff
@@ -18,7 +18,7 @@ if [ "$1" != "all" ] && [ "$1" != "frameworkonly" ]; then
 fi
 
 # Rename any stack_deletion out files if any
-for x in ~/jazz-installer/stack_de*.out
+for x in $JAZZ_INSTALLER_ROOT/stack_de*.out
```

Comment:
@bleggett Are you expecting $JAZZ_INSTALLER_ROOT set during installation time will retain whenever the user run destory.sh(manually).
User might face errors like this,
./destroy.sh: line 56: cd: /installscripts/jazz-terraform-unix-noinstances: No such file or directory

Reply:
1. $JAZZ_INSTALLER_ROOT should be set as an env var by `Installer.sh` in`run.py`, so it should be defined, but yes, it is possible that in the interim it might become unset for some reason, which leads to the larger problem:

2. Right now, it is pretty impossible to "back up" your installation, so that you can run `destroy` from a different machine. I plan to make `destroy.sh` much simpler, and remove most of the `JAZZ_INSTALLER_ROOT` references in it.

I want people to be able to save off/backup the `.tfstate` file, so that they can tear down the installation later on, even if the Centos box has been destroyed. This is the way Terraform is supposed to work: If you have the `.tfstate` file from a specific install, you can undo that install even from a different machine.

We can't do that, because `destroy.sh` is trying to do more than it needs to, so I want to make `destroy.sh` a "dumb" script that doesn't need any context or state to run.

With those changes, you could copy `.tfstate` and `destroy.sh` to a completely different box with Terraform installed (or zip them up and save them off for later), and tear down your Jazz install by running:

```
terraform destroy && destroy.sh $INST_NAME
```